js/edge-updates.js

```diff
@@ -52,10 +62,12 @@ STATUS = (function() {
         storage_set("total_pings", total_pings);
 
         var delay_hours;
-        if (total_pings == 1)      // Ping one hour after install
+        if (total_pings === 1) {      // Ping one hour after install
             delay_hours = 1;
-        else if (total_pings < 9)  // Then every day for a week
+        }
+        else if (total_pings < 9) {  // Then every day for a week
             delay_hours = 24;
+        }
```

Comment:
@kpeckett This ping code compares the latest version of CatBlock placed in a JSON file at cb.tk/edge.json with the actual installed version of CatBlock on Edge. When a newer version of CB for Edge has been detected, CB will show up a "New!" label on CB's logo.